src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/ReadType.java

```diff
@@ -0,0 +1,16 @@
+package io.stargate.sgv2.jsonapi.service.operation.model;
+
+/** Read type specifies what data needs to be returned as part of the response */
```

Comment:
maybe extend last part a bit: "part of the response for operations that read and return data"? Or something.
(not only for read operations as there's findOneAndModify and deleteOne that use this enum)